### ISSUE TITLE
MCP Email Copy Update #2

### ISSUE THREAD

**Issue by HeatherRienks** [department-of-veterans-affairs/va.gov-team#74795]
## Issue Description
Once we have received the approved verbiage of the new statement email for the medical copays (originally launched on June 6, 2023) we will need to add this to the existing VA Notify templates under the DMC service and add the template to our devops config.

For reference, you can refer to the previous MCP Epic #31490 and ticket #58189 may be helpful.

---
## Tasks
- [ ] Obtain final copy of new copay copy
- [ ] On VA Network login to VA Notify and create template in staging and prod
- [ ] Copy template IDs
- [ ] Add template ID to staging and prod devops settings

## Acceptance Criteria
- [ ] There exists a usable new copay email template to be referenced in API code
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by HeatherRienks** [comment]
Copy can be found here #61324

**Comment by HeatherRienks** [comment]
Per Kevin: The new email copy requires access to the amount outstanding on a given copay. That information should be stored in the data sent to our endpoint from VBS, but we don't have any example of the full data they send us. I requested an example so I could make a spec and go from there.

**Comment by kjsuarez** [comment]
VBS got back to me with an example of the data being sent to our `send_statement_notifications` endpoint and it does _not_ contain any data we could use to derive an outstanding balance. VBS will need to add this information to the parameters they send to our endpoint before we can update our email copy.

**Comment by denisecoveyduc** [comment]
I recommend we move forward with an interim solution for this ticket and create a new ticket for a future update.  What I mean is let's iteratively change this email notification as we are able to get more information to add additional clarity for Veterans.  

- **V1** - Current Live Version
- **V2** - (the update @kjsuarez is working on right now- as part of this ticket.)  We will change the first part of the copy to remove the copay $ amount. Instead the copy will read.....  " We're writing to inform you that as of January 2, 2024, you have an outstanding copay.  Please note that any payments you've ..."  (please reach out if there are any clarifying questions
- **V3**- (Future iteration) We will add $ amount, facility + copy changes (I have a few other copy changes we are iterating on now)
- **V4** - (Future iteration) Once we have payment history data we will look to update the copy again to provide additional clarity. 


@Tom-Davis or @HeatherRienks Would you be able to update the AC for this ticket to remove the copay $ amount and then establish ticket(s) for V3 and/or V4.  For cross referencing, if you can drop the links on this ticket that would be helpful Thank you.

**Comment by kjsuarez** [comment]
Back end updated to include personalization, VaNotify template updated to include copy and utilize personalization

**Comment by kjsuarez** [comment]
[PR](https://github.com/department-of-veterans-affairs/vets-api/pull/15491/files)
